### PR TITLE
FLA-490

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- update frontend demo and postman collection for generate url call
- use the passed in `data` and `md5` fields from demo app json
- jira: FLA-490 (https://justice.gov.bc.ca/jira/browse/FLA-490)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- locally

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by TayGov — APPROVED** [review]

File: src/backend/efiling-api/src/test/jag-efiling-api.postman_collection.json [hunk 1]
```diff
@@ -206,7 +206,7 @@
...
+              "raw": "{\n    \"navigationUrls\": {\n        \"success\": \"http//somewhere.com\",\n        \"error\": \"http//somewhere.com\",\n        \"cancel\": \"http//somewhere.com\"\n    },\n    \"clientAppName\": \"my app\",\n    \"filingPackage\": {\n        \"court\": {\n            \"location\": \"1211\",\n            \"level\": \"P\",\n            \"courtClass\": \"F\"\n        },\n        \"documents\": [\n            {\n                \"name\": \"test.pdf\",\n                \"type\": \"WNC\",\n                \"description\": \"Without Notice Application Checklist\",\n                \"statutoryFeeAmount\": 0,\n                \"data\": {},\n                \"md5\": \"string\"\n            }\n        ],\n        \"parties\": [\n            {\n                \"partyType\": \"IND\",\n                \"roleType\": \"CLA\",\n                \"firstName\": \"first\",\n                \"middleName\": \"middle\",\n                \"lastName\": \"last\"\n            }\n        ]\n    }\n}"
```
Comment (alexjoybc) [inline src/backend/efiling-api/src/test/jag-efiling-api.postman_collection.json:209]: Description does not exist I think, it's coming from old version of the api probably 🤣

File: src/frontend/efiling-demo/src/components/page/home/Home.js [hunk 2]
```diff
@@ -102,6 +104,8 @@ const generatePackageData = (files, filingPackage) => {
     documentData.push({
       name: files[i].name,
       type: document.type,
+      data: document.data,
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/src/components/page/home/Home.js:107]: 😜

**Review by alexjoybc — CHANGES_REQUESTED** [review]
Just an old thing in the postman payload to check....

File: src/frontend/efiling-demo/src/components/page/home/Home.js (hunk 2 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/efiling-demo/src/components/page/home/Home.js:107]: haha yes my blunder

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**